src/controllers/request.js

```diff
@@ -17,14 +19,49 @@ export default class RequestController {
    */
   static async getAllRequests(req, res) {
     try {
-      const allRequests = await Request.findAll({
-        where: {
```

Comment:
The where clause is intended to get all requests made by a PARTICULAR user to see all the requests that they have submitted over time to BareFoot Nomad

Corresponding PT Story: #167733825

Reply:
What action would you advice to be carried out? @EziamakaNV